src/Impostor.Server/Net/Inner/Objects/Components/InnerCustomNetworkTransform.Api.cs

```diff
@@ -6,6 +6,8 @@ namespace Impostor.Server.Net.Inner.Objects.Components
 {
     internal partial class InnerCustomNetworkTransform : IInnerCustomNetworkTransform
     {
+        public Vector2 Position => _targetSyncPosition;
```

Comment:
Is this correct? Isn't this still pointing to the target position?

Reply:
Yes, isn't _targetSyncPosition contains the player position?

Reply:
AFAIK _targetSyncPosition is the target position, not the current one.

Reply:
Yes, it is the target position, not the current player position. #170's PR was closed for using the same stuff.

Reply:
can I ask a question what is the target position?